src/main/java/edu/usf/cutr/tba/options/ProgramOptions.java

```diff
@@ -156,9 +161,16 @@ public  ProgramOptions setEndDate(long endDate) {
 
     public String getOutputDir() { return mOutputDir; }
 
-    public ProgramOptions setOutputDir (String outputDir) {
+    public ProgramOptions setOutputDir(String outputDir) {
         this.mOutputDir = outputDir;
         return this;
     }
 
+    public boolean getNoKMZ() { return mNoKMZ; }
```

Comment:
To make the code using this method easier to read you could name it something like `noKmz()`, or `skipKmz()` (see above). Also, Java conventions are for all uppercase values to only have the first letter capitalized - so in a method name `KMZ` should be `Kmz`.